spec/models/legal_aid_application_spec.rb

```diff
@@ -226,12 +226,6 @@
       end
       let(:da001) { legal_aid_application.proceedings.find_by(ccms_code: "DA001") }
       let(:se014) { legal_aid_application.proceedings.find_by(ccms_code: "SE014") }
-      let!(:chances_of_success) do
-        create(:chances_of_success, :with_optional_text, proceeding: da004)
-      end
-      let!(:chances_of_success) do
-        create(:chances_of_success, :with_optional_text, proceeding: se014)
-      end
```

Comment:
oh dear

Reply:
Yeah, I guess the test changed. We created two chance_of_success objects, neither is used and the da004 should have been 001! 😬 🤦‍♂️